unit_test/entities/test_production_cluster.py

```diff
@@ -198,12 +199,21 @@ def test_production_cluster_set_pressure_supply_negative(self) -> None:
     def test_production_cluster_write_to_output(self) -> None:
         """Test writing the output of a production cluster."""
         # Arrange
-        self.production_cluster.solver_asset.get_mass_flow_rate = Mock(return_value=1e6)
-        self.production_cluster.solver_asset.get_pressure = Mock(return_value=2e5)
-        self.production_cluster.solver_asset.get_temperature = Mock(return_value=333.15)
-
-        # Act
-        self.production_cluster.write_standard_output()
+        with (
```

Comment:
Dat ziet er netter uit. Hebben we dan nog wel de Mock nodig?

Reply:
Ja,  Mock wordt op andere plekken gebruikt.   Hier is dit iets netter,  de oude manier "verving" de methods van de solver_asset door Mock objects,  wat een typing/linting error triggert (assign-to-method).  Functioneel doet dit hetzelfde.